### PR TITLE
Relationship with no "data" member. Fixed #92

### LINKED ISSUE #92: Relationship without data property
Hi,
I'd like to create resource with following relationship `my_relation`, which only contains `links` property, ie. there is no `data` property with resource linkage:

```json
{
    "data": {
        "type": "...",
        "id": "...",
        "relationships": {
            "my_relation": {
                "links": {
                    "related": "/foo"
                }
            }
        }
    }
}
```

This is allowed as per [JSON:API spec](https://jsonapi.org/format/#document-resource-object-relationships): 

>  A “relationship object” MUST contain at least one of the following:
>     - links: a links object 
>     - data: resource linkage
>     - meta: a meta object that contains non-standard meta-information about the relationship.

Is there any way how to accomplish this now, with version 2 of the library? It was possible with version 1:
```php
$resourceObject->setRelationship('my_relation', Relationship::fromRelatedLink('/foo'));
```

However now there seems to be only `ToNull`, `ToOne` and `ToMany` relation, neither of which accomplishes exactly this (eg. ToNull adds `data: null` to the relationship).

### PR DESCRIPTION
**Pull request by f3ath**

### CODE REVIEW

**Comment by f3ath** [comment]
@OndraM could you please review this?

File: src/EmptyRelationship.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    /**
+     * @param object $o
+     */
+    public function attachTo($o): void
+    {
+        child($o, 'relationships')->{$this->name} = $this->obj;
+    }
+}
```
Comment (OndraM) [inline src/EmptyRelationship.php]: There is missing line-ending on the end of file (also in other new files)

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [2.0.2] - 2019-02-23
```
Comment (OndraM) [inline CHANGELOG.md]: It should be 2.1.0 according to SemVer (as new interface was added). I know this is "fix" from JSON-API point of view, but SemVer is from "code" point of view. 

https://semver.org/
> A bug fix is defined as an internal change that fixes incorrect behavior.

And while new interface can fix incorrect behavior, it is not "internal" change.

**Review by OndraM — COMMENTED** [review]
Some minor things, but overall looks good to me, thanks!

**Pull request merged**